build.gradle

```diff
@@ -18,46 +24,85 @@ repositories {
     jcenter()
```

Comment:
This jcenter() reference should be replaced with Maven Central (jcenter has been sunset as a central artifact repo). Comment can be removed as well